utils/envvars/parser.go

```diff
@@ -141,6 +148,38 @@ func parseStringDuration(value string) (time.Duration, error) {
 	return d, nil
 }
 
+// GetSAASHost returns SaaS host env variable value if it's present and valid.
+// Otherwise returns defaultSaaSHost.
+func GetSAASHost(fallbackEnv string) (string, error) {
+	name, v := envSaaSHost, os.Getenv(envSaaSHost)
+	if v == "" {
+		name, v = fallbackEnv, os.Getenv(fallbackEnv)
+		if v != "" {
+			logrus.Warnf("Environment variable %q WILL BE REMOVED SOON, please use %q instead.", fallbackEnv, envSaaSHost)
+		}
+	}
+
+	if v == "" {
+		logrus.Debugf("Using default SaaS host %q.", defaultSaaSHost)
+		return defaultSaaSHost, nil
```

Comment:
@BupycHuk @artemgavrilov I disabled nlreturn linter globally because it wanted add an empty line before returns, and that's, IMHO, worse